README.md

```diff
@@ -45,6 +45,9 @@ We have an ESP32 at the main shop entrance monitoring if it's open or closed, if
 
 Fetch latest team roster + payment status from OneDrive (TBD how), verify email, verify student number, verify student ID? If passing all requirements, the user is given the role "Verified".
 
+#### Onedrive sync
+Syncing to the onedrive team verification roster xlsx file is done through rclone + a cron task. Syncing is done every 5 minutes. To edit the synchronization file, please run `crontab -e` on the server to access the cron task configuration.
```

Comment:
```suggestion
#### OneDrive Sync - Danny
Syncing to the onedrive team verification roster xlsx file is done through rclone + a cron task. Syncing is done every 5 minutes. To edit the synchronization file, please run `crontab -e` on the server to access the cron task configuration.
```
This isn't detailed enough, if I came from a blank slate, I have no idea how this works except how to change how often it runs. Did you change what gets synced yesterday? Or does it still sync the entire drive instead of the administrative folder?
What's the server?
How do I set up the onedrive sync in the firstplace?
What OneDrive Syncing software are we using for linux?
This is incomplete.